config/aesgcm.go

```diff
@@ -78,3 +79,60 @@ func AesGcmDecrypt(ciphertext string, password string) (string, error) {
 
 	return string(plaintext), nil
 }
+
+func AesGcmEncryptWithKey(plaintext string, key []byte) (string, error) {
```

Comment:
we probably could use this function from the `AesGcmEncryptWithPassword` function? 
`AesGcmEncryptWithPassword` just also generates a key from the password, the rest is the same, isn't it